src/app/cartes/page.js

```diff
@@ -21,20 +22,21 @@ export default async function Cartes() {
 	return (
 		<>
 			<Nav />
-			<main className="min-h-screen w-full overflow-hidden py-8 pl-[calc(250px+4rem)] pr-16">
-				<div>
-					<h1 className="pb-2">Visualisation des cartes</h1>
+			{/* +4rem -> equivalent of pr-16 / pl-16  */}
+			<main className="flex min-h-screen w-full flex-col gap-6 overflow-hidden py-8 pl-[calc(250px+4rem)] pr-16">
```

Comment:
mettre un commentaire pour pas se poser la question du calc a chaque fois qu'on passe dessus 

```suggestion
			{/* 250px + 4rem -> espace pour ne pas passer sous la navbar     */}
			<main className="flex min-h-screen w-full flex-col gap-6 overflow-hidden py-8 pl-[calc(250px+4rem)] pr-16">
```

Reply:
done, bonne idée